GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -90,10 +90,10 @@ public void CanParseBackwardsDiff()
             diffBackwards.RequiredBlobs.Count.ShouldEqual(10);

```

Comment:
This test is not all that it could be...

1) It just tests counts.  It'd be better if it ToString'ed the filenames that are in each set (where possible) so that when it fails you have some idea of what changed.
2) It calls 'ParseDiffFile' rather than 'PerformDiff'.  This misled me into thinking that 'PerformDiff' really didn't have any tests.  'ParseDiffFile', as it turns out, is just called by tests.  It'd be better (and not hard  to do) to make it call 'PerformDiff'.

Wouldn't take much to twist my arm to do that.

Reply:
I wouldn't worry too much about that. I imagine it would be difficult to modify the code in a way that is incorrect and still gets these numbers correct.